spec/unit/network/resolver_spec.rb

```diff
@@ -14,45 +14,53 @@
 
     # The records we should use.
     @test_records = [
-      #                                  priority,  weight, port, hostname
+      #                                  priority,  weight, port, target
       Resolv::DNS::Resource::IN::SRV.new(0,         20,     8140, "puppet1.domain.com"),
       Resolv::DNS::Resource::IN::SRV.new(0,         80,     8140, "puppet2.domain.com"),
       Resolv::DNS::Resource::IN::SRV.new(1,         1,      8140, "puppet3.domain.com"),
       Resolv::DNS::Resource::IN::SRV.new(4,         1,      8140, "puppet4.domain.com")
     ]
+
+    @test_records.each do |rec|
+      # Resources do not expose public API for setting the TTL
+      rec.instance_variable_set(:@ttl, 3600)
+    end
   end
 
+  let(:resolver) { Puppet::Network::Resolver.new }
```

Comment:
Not expecting any change here, just recording some thoughts.

When I first saw this, I thought "Why not use the RSpec built-in `subject`?", but then I remembered that there's an `rspec-rubocop` [lint about having an explicitly named subject](http://rubocop-rspec.readthedocs.io/en/latest/cops_rspec/#rspecnamedsubject) (even though we're not using it), which means the only thing that would change here would be using `subject(:resolver)`, instead of `let(:resolver)`. Both would behave exactly the same, other than the `subject` form not leaving around the default `subject` method that would do the exact same thing as the `resolver` method.